protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1285,7 +1284,7 @@ enum AccountField {
 // AggregatedBalance data contains the balance of the set of requested accounts combined with corresponding
 // party ID, market ID and account type details
 message AggregatedBalance {
-  // Timestamp to of block the balance is referring to, in nanoseconds.
+  // Timestamp of the block, in which the balance exists (in nanoseconds).
```

Comment:
We're trying to avoid using `()` where possible. So maybe:
```suggestion
  // Timestamp in nanseconds of the block in which the balance exists.
```

Reply:
All right, that is corrected, thanks!